og.module

```diff
@@ -165,6 +165,13 @@ function og_entity_create_access(AccountInterface $account, array $context, $bun
 }
 
 /**
+ * Implements hook_cache_flush().
+ */
+function og_cache_flush() {
+  Og::groupResolverHandler()->updateConfigStorage();
+}
```

Comment:
I think probably the only case when we would need to update our config is when a new module that provides a plugin is enabled. So probably we can better do this on `hook_modules_installed()` instead of on every cache clear.